src/data/MockData.tsx

```diff
@@ -0,0 +1,44 @@
+import { faker } from "@faker-js/faker";
+
+type Comment = {
```

Comment:
Instead of redeclaring same attributes, consider using [Pick](https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys) where it makes sense (like here)

Reply:
Would that mean the commentText property from the type Comment be moved over to type User? 

Then using the Pick type to pick the ‘username’ and ‘commentText’ properties over to type Comment? 

type Comment = Pick<User, "username" | "commentText">;

export type User = {
  username: string;
  profilePic: string;
  commentText?: string;
};

Reply:
Or would this do?

type Comment = Pick<User, "username"> & { commentText: string };

Reply:
This question has more to do with business logic than typescript :)

I'd say the second suggestion

```ts
type Comment = Pick<User, "username"> & { text: string };
```